From 7853a753c47ae8c20dd51ce6d71b6d35a59c2c4e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mantas=20Mikul=C4=97nas?= <grawity@gmail.com>
Date: Fri, 18 Oct 2013 11:34:57 -0700
Subject: [PATCH 04/10] fix: Remove TLS version notification

The reason for this patch is that the "TLS connection" message adds an
unnecessary, annoying, unskippable delay to _every_ mutt startup.

(Almost as if its intention was to _discourage_ TLS usage...)
---
 mutt_ssl.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/mutt_ssl.c b/mutt_ssl.c
index 8cadc4f..44ee99b 100644
--- a/mutt_ssl.c
+++ b/mutt_ssl.c
@@ -444,14 +444,6 @@ static int ssl_negotiate (CONNECTION *conn, sslsockdata* ssldata)
   if (!ssl_check_certificate (conn, ssldata))
     return -1;
 
-  /* L10N:
-     %1$s is version (e.g. "TLSv1.2")
-     %2$s is cipher_version (e.g. "TLSv1/SSLv3")
-     %3$s is cipher_name (e.g. "ECDHE-RSA-AES128-GCM-SHA256") */
-  mutt_message (_("%s connection using %s (%s)"),
-    SSL_get_version(ssldata->ssl), SSL_get_cipher_version (ssldata->ssl), SSL_get_cipher_name (ssldata->ssl));
-  mutt_sleep (0);
-
   return 0;
 }
 
-- 
2.8.2

